Tools/unicode/makeunicodedata.py

```diff
@@ -26,13 +26,15 @@
 # written by Fredrik Lundh (fredrik@pythonware.com)
 #
 
+import dataclasses
 import os
 import sys
 import zipfile
 
+from dataclasses import dataclass
```

Comment:
Can we just qualify with `dataclass.` everywhere? (seems this is already mostly happening aside from the decorator application)